src/main/java/it/gov/pagopa/pu/classification/controller/AssessmentsDetailController.java

```diff
@@ -0,0 +1,28 @@
+package it.gov.pagopa.pu.classification.controller;
+
+import it.gov.pagopa.pu.classification.controller.generated.AssessmentsDetailApi;
+import it.gov.pagopa.pu.classification.dto.generated.CreateAssessmentsDetail;
+import it.gov.pagopa.pu.classification.model.AssessmentsDetail;
+import it.gov.pagopa.pu.classification.service.assessments.AssessmentsDetailService;
+import it.gov.pagopa.pu.classification.util.SecurityUtils;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@Slf4j
+public class AssessmentsDetailController implements AssessmentsDetailApi {
+    private final AssessmentsDetailService assessmentsDetailService;
+
+    public AssessmentsDetailController(AssessmentsDetailService assessmentsDetailService) {
+        this.assessmentsDetailService = assessmentsDetailService;
+    }
+
+    @Override
+    public ResponseEntity<List<AssessmentsDetail>> createAssessmentsDetail(Long organizationId, Long assessmentId, CreateAssessmentsDetail createAssessmentsDetail) {
+        log.debug("Create AssessmentsDetail having organizationId {} assessmentId {} request body {}",organizationId,assessmentId,createAssessmentsDetail);
```

Comment:
why debug level? there the risk of having too much log on this API? I would always use the info level on the controller
```suggestion
        log.info("Create AssessmentsDetail having organizationId {} assessmentId {} request body {}",organizationId,assessmentId,createAssessmentsDetail);
```